hera_mc/part_connect.py

```diff
@@ -292,6 +292,68 @@ def get_part_revisions(hpn, session=None):
     return revisions
 
 
+class Dubitable(MCDeclarativeBase):
+    """
+    A table to track the dubitable antennas.
+
+    start_gpstime:  start time of list
+    stop_gpstime:  stop time of list
+    ant_list:  list of "unapproved" antennas
+    """
+
+    __tablename__ = 'dubitable'
+
+    start_gpstime = Column(BigInteger, primary_key=True)
+    stop_gpstime = Column(BigInteger)
+    ant_list = Column(Text, nullable=False)
+
+    def __repr__(self):
+        return ('<{self.start_gpstime}>'.format(self=self))
+    #    return ('<{} dubitable entries>'.format(len(self.ant_list)))
+
+
+def update_dubitable(session=None, transition_gpstime=None, data=None):
+    """
+    update the database given a new list of dubitable antennas.
+    It will stop the previous list at transition_gpstime and start the new one
+    at transition_gpstime + 1sec
+
+    Parameters:
+    ------------
+    session:  db session to use
+    transition_gpstime:  gps time to make the change
+    data:  list of antennas
```

Comment:
This also needs to be more explicit about what `data` contains. Based on the code below, it looks like the list elements are expected to be integers? Can you add a cross-reference to the CM docs about what precise antenna identifier will be used here?